jsx/src/components/GroupEdit/GroupEdit.jsx

```diff
@@ -67,38 +102,50 @@ const GroupEdit = (props) => {
             id="submit"
             className="btn btn-primary"
             onClick={() => {
-              // check for changes
-              if (!changed) {
-                history.push("/groups");
-                return;
-              }
-
               let new_users = selected.filter(
                 (e) => !group_data.users.includes(e)
               );
               let removed_users = group_data.users.filter(
                 (e) => !selected.includes(e)
               );
-
               let promiseQueue = [];
-              if (new_users.length > 0)
-                promiseQueue.push(addToGroup(new_users, group_data.name));
-              if (removed_users.length > 0)
-                promiseQueue.push(
-                  removeFromGroup(removed_users, group_data.name)
-                );
+              // check for changes
+              if (changed) {
+                if (new_users.length > 0)
+                  promiseQueue.push(addToGroup(new_users, group_data.name));
+                if (removed_users.length > 0)
+                  promiseQueue.push(
+                    removeFromGroup(removed_users, group_data.name)
+                  );
+              }
 
+              if (hasDuplicates(propkeys) == true) {
```

Comment:
We should be showing errors above the input using the `.alert.alert-danger` bootstrap class. A feature making this easier is coming soon, but for the time being try using conditional blocks of JSX instead of editing properties.

```jsx
{ 
  isError
  ? <div className="error"></div>
  : <></>
}
```

Reply:
Done!